solr/core/src/test/org/apache/solr/cloud/ZkControllerTest.java

```diff
@@ -187,11 +187,11 @@ public void testGetHostName() throws Exception {
       } finally {
         if (zkController != null)
           zkController.close();
+        if (cc != null) {
```

Comment:
moving the declaration of `cc` to the assignment at line 179 would make the error here more clear. same for the other usage that you changed.